docs/assets/funfacts.txt

```diff
@@ -16,6 +16,21 @@ SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-
 You'll never believe this, but our Discord's support channels\nare in a category called SUPPORT IS HERE! Use them as necessary.
 SBE has skidded from countless mods including DSM, Skytils and NEU!
 SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
+In Canada, Santa's postal code is H0H 0H0.
+Zetvue makes music! You can too!
+Guvf vf abg na vzcbegnag fragrapr.
+ROT13
+Vs lbh qrpvcure guvf, lbh fubhyq yvfgra gb Mrgihr'f zhfvp.
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
given this is literal screaming you should make the first one as long as the second one

Reply:
the AAAAAAAAAA might go offscreen so thats why one is shorter than the other

Reply:
as in it's good if it's offscreen bc chaos

Reply:
```suggestion
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```